src/index.js

```diff
@@ -47,12 +47,16 @@ let ElasticsearchProvider = (config = { request: {} }) => {
           }
 
       let client = config.getClient()
+      let child = client.child({
```

Comment:
nit: Assuming client isn't referenced anywhere else, you could reduce the diff size with something like

```js
let child = config.getClient().child({ ....
```

Reply:
Sure I'll update that 👍